app/assets/javascripts/keyboard_tools.js

```diff
@@ -77,8 +77,27 @@ $(() => {
       }
     });
 
+    /**
+     * Checks common modifier states on a given keyboard event
+     * @param {KeyboardEvent} e
+     * @returns {boolean}
+     */
+    const getModifierState = (e) => {

```

Comment:
Perhaps `isModifierPressed` is a better name for this function?

Reply:
@Taeir probably, but I was trying to align the signature with the built-in API (which, unfortunately, lacks the required functionality of "is any modifier pressed?" as it can only check by key): https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState. I am not particular as to what to name it - if you, given the context, still think it's better to rename, will gladly do so!

Reply:
Ah, in that case I think this is fine :)